### DISCUSSION TITLE
Use image optimisation outside of the NextJS app

### DISCUSSION THREAD

**Discussion by akshaysmurthy** [vercel/next.js#23298] [Help]
We use `next/image` extensively in our web app for image optimisation. And I see that image URLs follow this pattern -  `https://{APP_HOST}/_next/image?url={ENCODED_SOURCE_IMAGE_URL}&w={WIDTH}&q={QUALITY}`

I couldn't find this URL pattern documented, but it does work with any source image URL as long as the domain is whitelisted.

What I am curious about now is whether I can build this URL outside of the Nextjs app. Eg., we have a browser extension that uses the same backend as our web app and we need to show the same images as part of the extension as well.

Has anybody done this?

**Comment by akellbl4** [comment]
I tried to optimize images with Next.js mechanism just using this pattern and everything works well.
You can found more options in `next/image` because first of all, it operates with all of the options that could be send to the Next.js image optimizer.
